src/stripe/LinkAuthenticationElement.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div>
+    <div id="link-authentication-mount-point"></div>
+  </div>
+</template>
+
+<script>
+import { loadStripe } from '@stripe/stripe-js';
+import { ref, onMounted } from 'vue';
+import {
+  STRIPE_PARTNER_DETAILS,
+  LINK_AUTHENTICATION_ELEMENT_TYPE,
+} from '../constants';
+export default {
+  props: {
+    pk: {
+      type: String,
+      default: undefined,
+      required: true,
+    },
+    stripeAccount: {
+      type: String,
+      default: undefined,
+    },
+    apiVersion: {
+      type: String,
+      default: undefined,
+    },
+    locale: {
+      type: String,
+      default: undefined,
+    },
+    disableAdvancedFraudDetection: {
+      type: Boolean,
+      default: false,
+    },
+    elementsOptions: {
+      type: Object,
+      default: () => ({}),
+    },
+  },
+  setup (props, { emit }) {
+    const stripe = ref(null);
+    const elements = ref(null);
+    const linkAuthElement = ref(null);
+
+    onMounted(async () => {
```

Comment:
Move everything inside that's inside of `onMounted` to a new function called `init` and invoke it instead.

```js
onMounted(() => {
	init();
});

async function init () {
 // code here
}
```

Reply:
ganto po ba?